### DISCUSSION TITLE
🎉 📅 v1.0 Launch Day Events!

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#1087] [Announcements]
![launch-event](https://user-images.githubusercontent.com/1509726/222925475-18825c51-bcce-4535-90f0-3dde15930efd.jpg)

Hey everyone, Chris here from the Skeleton crew 👋

After over a year in development, I'm thrilled to announce that [Skeleton](https://skeleton.dev/) will be officially launching v1.0 next week! 🎉

Unlike our typical release schedule, this will not drop on Tuesday, March 7th (US CST) as you might expect, but instead be delayed to Saturday, March 11th (US CST). This will give us just a little more time to polish things and allow the launch date to fall on Saturday (or Sunday, depending on your timezone).

This will come with a launch day event to celebrate both the project and the people responsible for making it happen. We'll be hosting not one but TWO live stream events this day, to help ensure folks around the world have a chance to participate.

## First Stream

- Saturday, March 11th
- Starts at noon (US CST)
- Length will be ~2-3 hours

## Second Stream

- Saturday, March 11th
- Starts around 8pm (US CST)
- Length will be ~2-3 hours

### What this will include:

- The launch of the v1.0 version of the library itself!
- The launch of our completely overhauled documentation website!
- An introduction to Skeleton - what it is, how it works, how to get started
- How to create a brand new Skeleton project
- Creating new custom themes and discuss general design tips and tricks
- Building new applications and general best practices
- We'll be answering your questions live on stream!
- And more!

### Where

Video the playlist here, with timestamps available in the video descriptions:
https://www.youtube.com/playlist?list=PLJtgPDPrJYAuDISpNrarLCjVInKknDCcA

Thanks again to everyone that's helped us reach this huge milestone. I appreciate you all! Please look forward to a very exciting launch event!

Best,
Chris

**Comment by BayoatHT** [comment]
Can't wait!

**Comment by jouwdan** [comment]
Congrats! Big milestone.

**Comment by RyanTheGuy** [comment]
Yee haw!

**Comment by endigo9740** [comment]
Please be aware that I've updated the original post with the link for either live stream on YouTube. Unfortunately due to technical difficulties we're going to have to cancel the Twitch stream. Sorry about that! Maybe we can do another even there later!

**Comment by endigo9740** [comment]
I've updated the original post, but here's the YouTube playlist of either livestream event, including timestamps in the video description:
https://www.youtube.com/playlist?list=PLJtgPDPrJYAuDISpNrarLCjVInKknDCcA
